docs/examples/use_cases/pytorch/efficientnet/image_classification/dataloaders.py

```diff
@@ -40,17 +38,19 @@
 DATA_BACKEND_CHOICES = ["pytorch", "synthetic"]
 try:
     from nvidia.dali.plugin.pytorch import DALIClassificationIterator
-    from nvidia.dali.pipeline import Pipeline
-    import nvidia.dali.ops as ops
     import nvidia.dali.types as types
 
-    DATA_BACKEND_CHOICES.append("dali-gpu")
-    DATA_BACKEND_CHOICES.append("dali-cpu")
-except ImportError:
+    from image_classification.dali import training_pipe, validation_pipe
+
+    DATA_BACKEND_CHOICES.append("dali")
+except ImportError as e:
     print(
         "Please install DALI from https://www.github.com/NVIDIA/DALI to run this example."
     )
 
+# TODO(klecki): Move it back again
+import torchvision.datasets as datasets
+import torchvision.transforms as transforms
```

Comment:
I don't know if this TODO is a leftover or something that should stay. However, if it should stay, I believe it should describe better what should be done ;)

Reply:
It doesn't matter much, but it breaks in my local setup as torchvision clashes with my local build of DALI, so I need to import DALI first.